aptos-move/framework/aptos-framework/sources/timestamp.move

```diff
@@ -59,6 +66,26 @@ module aptos_framework::timestamp {
         borrow_global<CurrentTimeMicroseconds>(@aptos_framework).microseconds
     }
 
+    /// Return the little-endian `u8`-vectorized time in microseconds,
+    /// useful as a form of quasi-nondeterminism.
+    public fun now_microseconds_vectorized():
```

Comment:
Since this can be front-run anyway, does it need to be so complicated? Can we just directly convert the timestamp to bytes?

Reply:
@movekevin timestamp serialization refactored per [e613916f62c55ad7d04de9e7198852ecad51ccaf](https://github.com/aptos-labs/aptos-core/pull/4173/commits/e613916f62c55ad7d04de9e7198852ecad51ccaf)